incubator/java-microprofile/image/Dockerfile-stack

```diff
@@ -30,12 +30,10 @@ ENV APPSODY_WATCH_REGEX="^.*(.xml|.java|.properties)$"
 
 ENV APPSODY_INSTALL="../validate.sh && mvn -B -Dmaven.repo.local=/mvn/repository install -DskipTests && /project/install-dev-deps.sh"
 
-ENV APPSODY_RUN="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -B -Dmaven.repo.local=/mvn/repository liberty:run"
-ENV APPSODY_RUN_ON_CHANGE="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -Dmaven.repo.local=/mvn/repository package -DskipTests"
+ENV APPSODY_RUN="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -B -Dmaven.repo.local=/mvn/repository liberty:dev -DskipTests"
```

Comment:
originally, the mp stack ran validate before each operation, this was flipped at somepoint to only run validate at init.
 
this means the parent pom is only being enforced when the project is cut from the template, which is kinda pointless, as you'd really think you could check that as part of the stack build, rather than at init time.. 

As-is, the end user of the project can just nuke the parent pom reference, change all the deps, liberty level etc, and nothing will prevent it..

Reply:
Good point. Will have to look at this. The developer experience whenever a code change is made is very slow if package is done every time. Removing the run on change allows just the app restart. Very nice and very quick. Will look into the run on change and see if it can be more selective as to when it does a package.

Reply:
Hadn't appreciated the validate routine and how it fit in.

Maybe we could prototype watching just the pom.xml and validating on a pom.xml change.. separately and off-to-the-side of LMP dev mode reacting to the POM change.  It could get a little confusing if you have two "processes" at once reacting to a single change, so let's try and see what that would look like.